src/Utils/Locator.php

```diff
@@ -56,6 +56,7 @@ public function getOwnCloudRootPath(){
 	 */
 	public function getRootDirContent(){
```

Comment:
I consider this approach quite fragile because we might want to add new files and folders into root - but this is topic of another discussion